crates/libeq_wld/src/parser/fragments/bsp_region.rs

```diff
@@ -374,21 +364,20 @@ impl BspRegionFragmentData4Entry {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug, PartialEq)]
 /// _Unknown_
-pub struct BspRegionFragmentData5Entry {
-    /// _Unknown_ - Usually 0.
-    params1: (u32, u32, u32),
-
-    /// _Unknown_ - Usually 0.
-    params2: u32,
+pub struct Obstacle {
+    /// bit 0 - is a FLOOR
+    /// bit 1 - is a GEOMETRYCUTTINGOBSTACLE
+    /// bit w - has USERDATA %s
```

Comment:
typo? should be bit 2?

Reply:
Yep, saw that last night 😄. I just pushed a fix and a bit more info.